### PR TITLE
Remove redundant parts of conditional expressions

### PR DESCRIPTION
**Pull request by alexandear**
I'm a member of the [Pinguem.ru](https://pinguem.ru) competition on finding errors in open source projects. All issues were found with [PVS-Studio](https://www.viva64.com/en/pvs-studio/):
V3022 Expression 'FileStatusListView.Items.Count > 0' is always true. FileStatusList.cs 861
V3022 Expression '(lineLength > 0)' is always true. EditNetSpell.cs 795
V3022 Expression 'revisions.Count > 2' is always true. FormFormatPatch.cs 141
V3022 Expression 'PushToRemote.Checked' is always true. FormPush.cs 470

### CODE REVIEW

File: GitUI/CommandsDialogs/FormPush.cs [hunk 1]
```diff
@@ -467,7 +467,7 @@ private bool HandlePushOnExit(ref bool isError, FormProcess form)
                 if (AppSettings.AutoPullOnPushRejectedAction == null)
                 {
                     bool cancel = false;
-                    string destination = PushToRemote.Checked ? _NO_TRANSLATE_Remotes.Text : PushDestination.Text;
```
Comment (gerhardol) [inline GitUI/CommandsDialogs/FormPush.cs]: While the rewrite is OK, is this really what is desired?

File: GitUI/UserControls/FileStatusList.cs [hunk 2]
```diff
@@ -858,8 +858,7 @@ public void SelectFirstVisibleItem()
                 if (sortedFirstGroupItem != null)
                     sortedFirstGroupItem.Selected = true;
             }
-            else if (FileStatusListView.Items.Count > 0)
-                FileStatusListView.Items[0].Selected = true;
+            FileStatusListView.Items[0].Selected = true;
```
Comment (gerhardol) [inline GitUI/UserControls/FileStatusList.cs]: Does not this mean that group==null always?

File: GitUI/CommandsDialogs/FormPush.cs (hunk 1 above)
Comment (alexandear) [reply GitUI/CommandsDialogs/FormPush.cs]: Yes.

**Comment by alexandear** [comment]
@gerhardol fixed

**Comment by RussKie** [comment]
Thank you. I've read about the PVS some while ago, it sounded like a very cool thing.
To be honest I'm suprised this all you have found

**Pull request merged**